rules.md

```diff
@@ -4,74 +4,79 @@ title: APIO 2021 Competition Rules
 key: rules
 ---
 
-These Competition Rules cover General, Problem Scoring, Contestant Scoring, and Clarifications in APIO 2021.
-Delegation Leaders have the responsibility of ensuring that all members of their delegation fully understand these rules and abide by them.
+The following are the APIO 2021 Competition Rules. Delegation Leaders have the responsibility of ensuring that all members of their delegation fully understand these rules and abide by them.
 
-<span style="color:red">**Please do not share the problems after the contest within 2 days since the competition is not held within the same time range.**</span>
+<span style="color:red">**Please do not share the tasks after the contest within 2 days since the competition is not held within the same time range.**</span>
 
 <br>
 
 ### A. General
 
 * APIO 2021 consists of **1 practice problemset** and **1 competition day**.
 * Result of the practice problemset **does not** affect scoring.
-* There will be **3 (three) problems** on the competition day to be solved in **5 (five) hours**.
-* Each contestant may submit up to **50 submissions** for each problem.
+* There will be **3 (three) tasks** on the competition day to be solved in **5 (five) hours**.
 * The supported programming language is **C++**.
 * During the competition, each contestant can only see their own scores.
 
 <br>
 
-### B. Problem Scoring
+### B. Submission
 
-* There are two types of scoring: **standard** and **creative**.
-* Creative problems will be stated explicitly in the problem description.
-
-* **Standard** scoring:
-  * For batch and interactive problems:
-    * A problem consists of **multiple subtasks** with various points.
-    * A subtask consists of **multiple test cases**.
-    * A submission gets a subtask's points if it solves **all of its test cases**.
-  * For output-only problems:
-    * A problem consists of **multiple test cases** with various points.
-    * A submission gets the test case's points if it produces the correct output.
-
-* **Creative** scoring:
-  * The scoring formula will vary for each problem and will be explicitly stated in the problem description.
+* The source program provided by the contestant must be contained in one source file as specified in the task statement.
+* Submissions must not perform explicit input and output operations; instead data must only be exchanged through the interfaces specified in the task statement.
+* For each programming task, the contestants can download a zip file for a programming task from the grading system.
+  * The zip file contains interface files, a sample grading program, and a skeleton implementation of a required source file.
+  * The skeleton exercises an interface, but it does not solve the task.
+  * The provided sample grader would not be the same as the official grader used by the grading system.
+* Each contestant may submit up to **50 submissions** for each task.
```

Comment:
```suggestion
* Each contestant may submit up to **50 submissions** for each task. This limit may be increased for some task types.
```

ini perlu ga?

Reply:
Gw tambahin "unless otherwise stated in the task statement" instead.